ext/sodium/config.w32

```diff
@@ -0,0 +1,15 @@
+// $Id$
+// vim:ft=javascript
+
+ARG_WITH("libsodium", "for libsodium support", "no");
```

Comment:
The ext name should be adjusted here, too. In further also the variable would be PHP_SODIUM. I'm going to add the library builds to the windows deps package already.

Thanks.